go/vt/vttablet/tabletserver/tabletserver.go

```diff
@@ -228,7 +228,7 @@ func NewTabletServer(name string, config *tabletenv.TabletConfig, topoServer *to
 // to complete the creation of TabletServer.
 func (tsv *TabletServer) InitDBConfig(target querypb.Target, dbcfgs *dbconfigs.DBConfigs, mysqld mysqlctl.MysqlDaemon) error {
 	if tsv.sm.State() != StateNotConnected {
-		return vterrors.Errorf(vtrpcpb.Code_UNKNOWN, "InitDBConfig failed, current state: %s", tsv.sm.IsServingString())
+		return mysql.NewSQLError(mysql.ERServerIsntAvailable, mysql.SSUnknownSQLState, "Server isn't available")
```

Comment:
instread of creating mysql errors, create error with vterrors.NewErrorf

Reply:
👍